src/clr-angular/data/tree-view/models/tree-node.model.spec.ts

```diff
@@ -16,7 +16,7 @@ class TestModel extends TreeNodeModel<string> {
     this.parent = parent;
   }
 
-  children: TestModel[];
+  children: TestModel[] = [];
```

Comment:
Please remove this once you add the `if (this.children)` I mentioned in a previous comment. Our tree nodes are supposed to work fine with a `null` or `undefined` list of children.